app_dart/lib/src/service/branch_service.dart

```diff
@@ -35,9 +35,13 @@ class BranchService {
       return;
     }
     final String? branch = createEvent.ref;
-    final String? repository = createEvent.repository!.slug().fullName;
+    final String repository = createEvent.repository!.slug().fullName;
     final int lastActivity = createEvent.repository!.pushedAt!.millisecondsSinceEpoch;
 
+    if (repository.split('/')[0] != 'flutter') {
```

Comment:
We may want to support future orgs like dart-sdk. Is there a way we can avoid hard coding this to only support Flutter?

Reply:
yes sir